commons/src/main/java/org/eclipse/kapua/commons/setting/system/SystemSettingKey.java

```diff
@@ -340,6 +340,14 @@ public enum SystemSettingKey implements SettingKey {
      * Provide the Cache TTL
      */
     CACHE_TTL("commons.cache.config.ttl"),
+    /**
```

Comment:
In the previous PR it was already pointed out that at least one of these properties is already present (https://github.com/eclipse/kapua/pull/3804#discussion_r1250621583) . I'd suggest to either use that, or remove the old one in favour of there new ones.

Reply:
I thought I used the old one as you suggested, I'll double check the code thanks!